src/HealthGPS.Input/model_parser.cpp

```diff
@@ -268,15 +273,15 @@ load_staticlinear_risk_model_definition(const nlohmann::json &opt, const Configu
         Eigen::MatrixXd{Eigen::LLT<Eigen::MatrixXd>{policy_covariance}.matrixL()};
 
     // Risk factor expected values by sex and age.
-    hgps::RiskFactorSexAgeTable expected = load_risk_factor_expected(config);
+    std::unique_ptr<hgps::RiskFactorSexAgeTable> expected = load_risk_factor_expected(config);
```

Comment:
I'd generally use `auto` in place of longer type names:

```suggestion
    auto expected = load_risk_factor_expected(config);
```

But up to you.